src/ralph/virtual/management/commands/openstack_sync.py

```diff
@@ -361,6 +368,75 @@ def _get_hypervisor(host_name, server_id):
             ))
             return None
 
+    def _match_physical_and_cloud_hosts(self):
+        """Connect CloudHosts and DC assets according to data from Ironic."""
+
+        provider = getattr(
+            settings, 'OPENSTACK_IRONIC_PROVIDER', 'openstack-ironic'
+        )
+        os_conf = next((conf for conf in settings.OPENSTACK_INSTANCES
+                        if conf['provider'] == provider), None)
+
+        if os_conf is None:
+            logger.error('Ironic is not configured.')
+            return
+
+        ironic_client = get_ironic_client(
+            api_version=os_conf['version'],
```

Comment:
If that's an ironic configuration, I suppose, it will provide Ironic API version.